src/workspace-data/data-table/entity-service/renderDataCell.js

```diff
@@ -45,6 +45,12 @@ export const renderDataCell = (attributeValue, workspace) => {
   } = workspace;
 
   const renderCell = (datum) => {
+    // handle nested arrays, where datum is itself an array. Recurse back to renderArray() for this, but surround its
+    // result with [] to indicate to the end user where the nested arrays start and stop
+    if (_.isArray(datum)) {
+      return span({}, ['[', renderArray(datum), ']']);
+    }
```

Comment:
the use of a `span` wrapper here ensures unique key values within `renderArray`; each invocation of `renderArray` is contained in a dedicated span.